system-parachains/asset-hubs/asset-hub-polkadot/src/lib.rs

```diff
@@ -435,6 +436,18 @@ impl pallet_utility::Config for Runtime {
 	type WeightInfo = weights::pallet_utility::WeightInfo<Runtime>;
 }
 
+parameter_types! {
+	pub const IndexDeposit: Balance = DOLLARS / 10; // 10 Dollar on the Relay Chain
```

Comment:
I think a new deposit can be just calculated with `system_para_deposit` function